src/App.jsx

```diff
@@ -28,6 +29,7 @@ function App() {
         <Routes>
           <Route path="/" element={<Home articles={articles}/>}/>
           <Route path="/all-articles" element={<AllArticles articles={articles}/>}/>
+          <Route path={`/all-articles/:article_id`} element={<Article/>}/>
```

Comment:
Nice addition of a Route with a parametric endpoint, for the path you wouldn't need curly braces around it if you didn't use backticks